addons/bestja_page_fixtures_ucw/templates/pages/become_volunteer.xml

```diff
@@ -117,19 +117,19 @@
                                 <div class="col-md-8 col-md-offset-2">
                                     <ul class="like_write">
                                         <li class="mb64">
-                                            <span>
```

Comment:
Thanks to Ludwik, we discussed the matter of the URL. The thing is, that we need Like URL, not only Facebook address: https://www.facebook.com/UCWUW. Could you respond to this comment?